adafruit_is31fl3731.py

```diff
@@ -93,7 +93,6 @@ def __init__(self, i2c, address=0x74):
         self.i2c = i2c
         self.address = address
         self._frame = None
-        self.reset()
```

Comment:
If you're removing a function call, delete the function on L154.

Reply:
Though, I'd prefer if the public `reset` method wasn't removed from this library and instead you could refactor the initialization routine and/or `reset`

Reply:
I don't think I understand what you mean. Just because the initialization sequence is fixed doesn't mean that a public reset method is not useful. How would you "refactor" the code so you still have the public reset method, but still have the initialization fixed?

Reply:
I think this is fine. In newer libraries reset is usually `deinit` and then recreate the driver object. However, this library predates it and is fine as is.